src/groups/mqb/mqba/mqba_sessionnegotiator.cpp

```diff
@@ -146,6 +146,17 @@ void loadBrokerIdentity(bmqp_ctrlmsg::ClientIdentity* identity,
             .append(bmqp::MessagePropertiesFeatures::k_FIELD_NAME)
             .append(":")
             .append(bmqp::MessagePropertiesFeatures::k_MESSAGE_PROPERTIES_EX);
+
+        const mqbcfg::AppConfig& theConfig = mqbcfg::BrokerConfig::get();
+
+        if (theConfig.brokerVersion() == 999999 ||
+            (theConfig.configureStream() &&
+             theConfig.advertiseSubscriptions())) {
+            features.append(";")
```

Comment:
Is the trigger to enable subscription to append this "SUBSCRIPTIONS:CONFIGURE_STREAM"?

Reply:
Yes!

Reply:
We need to add this to https://github.com/bloomberg/blazingmq/pull/224